### PR TITLE
fix(website examples): stats widget issue

### PR DESCRIPTION
**Pull request by morphoix**
1. Width of examples from
Screenshot from 2023-04-21 19-09-21
to
Screenshot from 2023-04-21 19-09-55

2. Stats widget issue (I3s and 3dTiles)
3. 3dTiles: Info panel text color (dark mode)
Screenshot from 2023-04-21 19-08-10

Screenshot from 2023-04-21 19-09-01

### CODE REVIEW

**Review by ibgreen — APPROVED** [review]
@morphoix This is amazing, bug after bug is being fixed, so quickly that I hardly have time to keep up with reviews. Very impressive!

File: examples/website/3d-tiles/app.tsx [hunk 1]
```diff
@@ -35,6 +35,21 @@ export const INITIAL_VIEW_STATE = {
   zoom: 3 // Start zoomed out on US, tileset will center via "fly-to" on load
 };
 
+const Stats = styled.div`
```
Comment (ibgreen) [inline examples/website/3d-tiles/app.tsx:38]: Ideally we have better defaults in the stats widget itself.

Comment (morphoix) [reply examples/website/3d-tiles/app.tsx:38]: The problem is that default position set to 'fixed' as I figured and somehow the whole docusaurus container takes this as a rule, not the parent position even when it sets through ref container, so the solution with overwriting styles without logic changing was quick :)

Comment (morphoix) [reply examples/website/3d-tiles/app.tsx:38]: as I found in docs in probe.gl we can overwrite css for header or info, but not for all container

**Pull request merged**